tests/test_js_graph.py

```diff
@@ -444,6 +463,11 @@ def test_insert_child_table(graph_cases):
     tables = graph.get_tables()
     assert len(tables) == 2
 
+    # After resetting the zoom, bounds must be the expected
```

Comment:
"expected" as in `expected_scale`?

Reply:
`coords = {x: x, y: y};` means to use regular x, y coords instead of adjusted.

"bounds must be the expected" is about the actual relevant tests of child table bounds, at the end. I meant to explain why we are applying zoom, as in "the table bounds must respect those constraints even when some zoom is applied".

Reply:
I see, thanks. I suggest to update the comment, your explanation was much clearer. 👍

Reply:
Similar to a python `{'x': x, 'y': y}`, this object has the same structure as the one returned by `graphs.utils.adjustCoordinates`.